hubblestack/extmods/module_runner/runner.py

```diff
@@ -169,38 +169,46 @@ def _is_hubble_version_compatible(self, profile_id, yaml_dictionary_data):
         """
         log.debug("Current hubble version: %s" % __grains__['hubble_version'])
         current_version = version.parse(__grains__['hubble_version'])
-        version_str = yaml_dictionary_data.get('hubble_version', '')
+        version_str = yaml_dictionary_data.get('hubble_version', '').strip()
         if not version_str:
             log.debug("No hubble version provided for check id: %s Thus returning true for this check" % (profile_id))
             return True
+        if '*' in version_str:
```

Comment:
Regex can have many more characters apart from '*'.
I think we can leave this handling to version.parse(). If it is able to parse it, its valid. Else not.

Reply:
Yes regex can have many chars but version.parse don't return error in case it is unable to parse a version. It simply returns a string. So can't rely on it.